python/protobuf_rpc/base_server.py

```diff
@@ -17,33 +22,52 @@ def run(self, response):
         self.invoked = True
 
 
+LOG_FORMAT = ''
+access_log = getLogger('protobuf_rpc.access')
+
+
 class ProtoBufRPCServer(object):
+    def __init__(self, log_format=LOG_FORMAT):
+        self.log_format = log_format
+
     def handle(self, request):
+        start = time.time()
         try:
             req_obj = self.parse_outer_request(request)
         except Exception as e:
             return self.build_error_response(e.message, INVALID_REQUEST_PROTO)
+        try:
+            logging_params = {'method': req_obj.method_name, 'request': json.dumps(protobuf_to_dict(req_obj))}
+        except Exception as e:
+            access_log.exception("Error writing to logging params")
```

Comment:
We always want to write a single, consistent format to the access log to enable machine reading. Make a different logger for errors.